src/projection/equalEarth.js

```diff
@@ -19,11 +18,11 @@ export function equalEarthRaw(lambda, phi) {
 
 equalEarthRaw.invert = function(x, y) {
   var l = y, l2 = l * l, l6 = l2 * l2 * l2;
-  for (var i = 0, dl, fy, fpy; i < ITERATIONS; ++i) {
+  for (var i = 0, delta, fy, fpy; i < iterations; ++i) {
     fy = l * (A1 + A2 * l2 + l6 * (A3 + A4 * l2)) - y;
     fpy = A1 + 3 * A2 * l2 + l6 * (7 * A3 + 9 * A4 * l2);
-    l -= dl = fy / fpy, l2 = l * l, l6 = l2 * l2 * l2;
-    if (abs(dl) < EPSILON) break;
+    l -= delta = fy / fpy, l2 = l * l, l6 = l2 * l2 * l2;
+    if (abs(delta) < epsilon2) break;
```

Comment:
What I meant was did you assess (empirically) whether changing from 1e-9 to 1e-6 or 1e-12 would impact performance or accuracy? I don’t know how quickly this converges, so I don’t know whether this loop breaks because the accuracy is deemed sufficient or because the maximum number of iterations is reached.

Alternatively, we could just stick with 1e-9 from the reference implementation.

Reply:
Yes (sorry I wasn't clear): I used `console.warn("iterations", i)` to show the number of steps with epsilon2 = 1e-12 — it converges in 3 iterations for the test points in tests/projection/invert-test.js (and a few others I added locally, but that break other projections). Even for point=[179.99,89.9] we only use 4 iterations to get to that delta.